roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -144,3 +144,32 @@ pyclean () {
         find . -type d -name "__pycache__" -delete
 }
 _pyclean_help="Cleanup extra python files"
+
+pbindings(){
```

Comment:
Does making some way to `install`, `uninstall`, or `reinstall` make any sense. As code changes in PRs it would be convenient to have 1 command to reinstall that plugins' bindings. What do you think?

Reply:
the command is idempotent, so for reinstall, you just need to call it again

Reply:
oh nice that makes it simple which is great. No changes needed here.